frontend/amundsen_application/static/js/config/config-default.ts

```diff
@@ -385,7 +385,6 @@ const configDefault: AppConfig = {
     isEnabled: false,
   },
   nestedColumns: {
-    isEnabled: true,
```

Comment:
Are there any documentation that we need to update due to the removal of this option?

Reply:
There is not, I referred to the original PRs for the addition of these changes to make sure I covered all the parts to be removed, and didn't come across any related docs